nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -23,6 +23,23 @@ in
           If left at the default value, $HOME/.i3/config will be used.
         '';
       };
+
+      debug = mkOption {
+        default = false;
+        example = true;
+        type = types.bool;
+        description = "Enable debug/verbose logging (see -V option)";
+      };
+
+      logFile = mkOption {
+        default = null;
+        example = "$HOME/.i3/i3log";
```

Comment:
@jagajaga This does not help at all. What if I want different users to have different logfiles ? Shoul I use ${config.users.extraUsers.foo.home}/.i3/i3log or ${config.users.extraUsers.bar.home}/.i3/i3log ? It makes sense for each user to have the logs of its own window manager instance in its own home, no ?